packages/SwingSet/test/vat-env.test.js

```diff
@@ -97,13 +97,18 @@ async function testForExpectedGlobals(t, workerType) {
     'VatData.makeScalarBigWeakMapStore: function',
     'VatData.makeScalarBigSetStore: function',
     'VatData.makeScalarBigWeakSetStore: function',
+    'global has passStyleOf: true',
+    'global passStyleOf is special: false',
   ]);
 }
 
 test('expected globals are in the local worker vat environment', async t => {
   await testForExpectedGlobals(t, 'local');
 });
 
-test('expected globals are in the XS worker vat environment', async t => {
-  await testForExpectedGlobals(t, 'xs-worker');
-});
+test.failing(
```

Comment:
This PR so far only fixes the node case. For undiagnosed reasons, the XS case still fails.

Reply:
False alarm. The XS case seems to work. Resolving.